pages/_app.tsx

```diff
@@ -3,7 +3,7 @@
 // Polyfills
 import 'intersection-observer';
 
-import { useEffect } from 'react';
+import { useEffect, type PropsWithChildren } from 'react';
```

Comment:
```suggestion
import { useEffect } from 'react';
import type { PropsWithChildren } from 'react';
```

Reply:
When you enable TypeScript ESLint as requested in a later comment, [this rule](https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/docs/rules/consistent-type-imports.md) (in it's default config) will automate this distinction.

Reply:
Would you like me to change the parser from `@babel/eslint-parser` to `@typescript-eslint/parser` too?

Reply:
We can do 2 sets of parsers! I'd keep `@babel/eslint-parser` for JS files and the TS ESLint parser for TS.